src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './Tabs.scss';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import classNames from 'classnames';
+
+type Props = {
+  tabs:Tab[]
+  selectedTabId:string
+  onTabSelect:(tabId:Tab) => void
```

Comment:
ты передаешь объект, но переменная называется tabId
это некорректно
```suggestion
  onTabSelect: (tabId: Tab) => void
```